Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "jsonwebtoken"
-version = "6.0.1"
+version = "6.0.2"
```

Comment:
that's the issue, updating ring is a breaking change so that would have to be v7